TeamCode/src/main/java/org/firstinspires/ftc/teamcode/Robot/Systems/SpampleArm.java

```diff
@@ -12,10 +12,17 @@ public class SpampleArm {
     ServoPlus twistServo;
     Claw claw;
 
+    double targetExtension = 0;
+    
     //TODO: replace with correct value; calibrated for 312 RPM motor
+    //TODO: the motor will not turn correctly without these values right.
     //Constants for the shoulder
     double shoulderPulsesPerRevolution = 537.7;
```

Comment:
I think this should be a final variable if it is a constant (there are a few other constant variables in this code which also need to be changed to final).